submissions/innazozulia/DOM practice/main.js

```diff
@@ -0,0 +1,153 @@
+const Document = document;
```

Comment:
No need to make this reassignment. Just use `document` in when you query some elements